website/docs/guides/migration/versions/upgrading-to-v1.2.md

```diff
@@ -15,9 +15,105 @@ dbt Core v1.2 will soon be available as a **beta prerelease.** Join the #dbt-pre
+
+As for how to make it happen, looking at the following PRs for dbt-Labs-maintained adapters show it clearly:
+
+- [dbt-bigquery#192](https://github.com/dbt-labs/dbt-bigquery/pull/192) 
+- [dbt-redshift#120](https://github.com/dbt-labs/dbt-redshift/pull/120) 
+- [dbt-snowflake#162](https://github.com/dbt-labs/dbt-snowflake/pull/162) 
+
+
+### Grants
+
+Managing access grants are one of the most asked for features from dbt users. We’re delivering this capability, but naturally there’s variance across data platforms as to how grants work, so time for adapter maintainers to roll their sleeves up. You might get lucky and not have to override any of them, but in case you do, below are descriptions of the new methods and macros, grouped into level of complexity (start with the easy ones first!)
+
+:::info
+this new functionality does not add users, only grants access. You'll have to handle that elsewhere, and it has [implications for the GRANT adapter tests](#testing-grants-for-your-adapter).
+:::
+
+PRs for adding grants for dbt Labs maintained adapters should be very useful as a reference, e.g. [dbt-bigquery#212](https://github.com/dbt-labs/dbt-bigquery/pull/212).
+
+#### Overrideable macros and methods
+
+The two macros below are simple Boolen-toggles (i.e. `True/False` value) indicating whether certain features are available for your database. The default of both of these macros are `True`, because we believe that all databases should support these ergonomic features. However, we've built for flexibility, so overriding these macros for your adapter, will handle the case where your database doesn't support these features.
+
+| macro | description | global project’s default | example override |
+| --- | --- | --- | --- |
+| `copy_grants()` | when an object is fully replaced on your database, do grants copy over? e.g. on Postgres this is never true, on Spark this is different for views vs. non-Delta tables vs. Delta tables, on Snowflake it depends on the user-supplied `copy_grants` configuration. true by default, which means “play it safe”: grants MIGHT have copied over, so dbt will run an extra query to check them + calculate diffs. | [`default__copy_grants()`](https://github.com/dbt-labs/dbt-core/blob/c25260e5dd2afa237a30db115605ece9629443d1/core/dbt/include/global_project/macros/adapters/apply_grants.sql#L3-L21)| [`snowflake__copy_grants()`](https://github.com/dbt-labs/dbt-snowflake/blob/d53c327e20c91522b4792ede75bbe50e16a9d9c3/dbt/include/snowflake/macros/adapters.sql#L297-L300) |
+| `support_multiple_grantees_per_dcl_statement()` | does this database support `grant {privilege} to user_a, user_b, ...`? or do `user_a` + `user_b` need their own separate grant statements? | [`default__support_multiple_grantees_per_dcl_statement()`](https://github.com/dbt-labs/dbt-core/blob/c25260e5dd2afa237a30db115605ece9629443d1/core/dbt/include/global_project/macros/adapters/apply_grants.sql#L24-L39) |  |
```

Comment:
@runleonarun any idea on how to make the column width not suck like it does right now?
![image](https://user-images.githubusercontent.com/8158673/178551401-10b1dea1-5ba7-4dc9-8255-d3fd846803f8.png)